packages/core/src/PartialExecutionUtils/DirectedGraph.ts

```diff
@@ -470,6 +470,12 @@ export class DirectedGraph {
 		return graph;
 	}
 
+	static fromDirectedGraph(graph: DirectedGraph) {
```

Comment:
Nit - `toX` and `fromX` imply transformation or?
```suggestion
	static cloneDirectedGraph(graph: DirectedGraph) {
```

Reply:
Don't have a strong opinion on this. I'll change it.

Reply:
Maybe just `clone` and then as an instance method? That's the other option I had in mind.

I think I'll do that.